package.json

```diff
@@ -14,8 +14,9 @@
   },
   "devDependencies": {
     "electron-mocha": "^6.0.0",
-    "mocha": "^5.0.0",
     "jsdoc": "^4.0.0",
+    "mocha": "^5.0.0",
+    "node-gyp": "^9.4.0",
```

Comment:
For whatever reason, running Prebuildify kept trying to find a local copy of `node-gyp` instead of using my globally installed one